### DISCUSSION TITLE
Equivalent nrpe check_counter plugin in NCPA

### DISCUSSION THREAD

**Discussion by jriverastyle** [community/community#69376] [Programming Help]
### Select Topic Area

General

### Body

Hi,
Im trying to add monitoring of windows related counters like IIS and asp .net performance counter in Nagios. When using nrpe/nsclient there is this plugin check_counter that can be used to retrieve windows counters in the remote windows server. Is there an equivalent plugin when using ncpa?
Thank you!

**Comment by samus-aran** [comment]
Welcome to the GitHub Community, @jriverastyle , we're happy you're here! You are more likely to get a useful response if you are posting your question(s) in the applicable category and are explicit about what your project entails--giving a few more details might help someone give you a nudge in the right direction. I've gone ahead and moved it for you. Good luck!
